share/change.py

```diff
@@ -250,11 +255,12 @@ def is_blank(self):
 
     @property
     def is_skippable(self):
-        if self.instance and self.graph.source and hasattr(self.instance, 'sources'):
-            new_source = not self.instance.sources.filter(source=self.graph.source).exists()
-        else:
-            new_source = False
-
+        new_source = False
+        if self.instance:
+            if self.type == 'subject' and self.instance.central_synonym is None and self.graph.source.user.username != settings.APPLICATION_USERNAME:
```

Comment:
Yeah, if it's a Bepress subject (`central_synonym is None`) from a user besides `system`, this just silently ignores any changes.